.github/workflows/wheels.yml

```diff
@@ -103,12 +115,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ 'ubuntu-latest', 'windows-latest' ]
+        os: [ 'ubuntu-latest', 'windows-latest', 'macos-latest' ]
         python-version: ['3.7', '3.8', '3.9','3.10']
         arch: ['x64', 'x86']
         exclude:
         - os: 'ubuntu-latest'
           arch: 'x86'
+        - os: 'macos-latest'
+          arch: 'x86'
 
```

Comment:
Just realised that we are not testing aarch64 (linux), this is not related to this specific PR anyways.

Reply:
we are not and we really cannot realistically.

github is not offering ARM based runners at the moment, and running tests with emulation (qemu + docker) is pretty slow (30+ minutes to run the "light" test-suite when i last ran that).

Reply:
> running tests with emulation (qemu + docker) is pretty slow (30+ minutes to run the "light" test-suite when i last ran that)

OK you convinced me
This is something that can be implemented in the future when proper runners will be available.
I will file in issue for that as a reminder.